lib/strategy.js

```diff
@@ -45,12 +46,13 @@ function Strategy(options, verify) {
     options = {};
   }
   if (!verify) { throw new TypeError('LocalStrategy requires a verify callback'); }
-  
+
   this._usernameField = options.usernameField || 'username';
   this._passwordField = options.passwordField || 'password';
-  
+  this._totpField = options.totpField || 'totp';
```

Comment:
These strategies allow you to customize the names of the form fields that contain the required fields. I call this simply `totp` by default, but our frontend implementation may choose to call it e.g. `otp_attempt` to align with Gravity.